index.html

```diff
@@ -12,6 +12,7 @@ <h1>Catch the Cat! It's so fun!</h1>
         <p>Score: <span id="score">0</span></p>
         <div id="cat" class="cat"></div>
         <button id="start-button">Start Game</button>
+        <button id="pause-button">Pause Game</button>
```

Comment:
looks like styling for the button wasn't implemented. Here is the current state of the game when I test it. Did you test it before opening a PR?

![Screenshot 2024-08-12 at 4 49 46 PM](https://github.com/user-attachments/assets/f24fd534-1956-4e16-b64e-9068dd1a89fa)

Reply:
Im seeing the same on my side, the button isn't styled

Reply:
This is now updated!

![Screenshot 2024-08-12 at 4 55 43 PM](https://github.com/user-attachments/assets/5ff54095-ce09-4dac-beea-cd4018b707e9)